tests/framework/installer/ceph_manifests.go

```diff
@@ -328,9 +334,9 @@ parameters:
   pool: ` + poolName + `
   clusterID: ` + m.settings.Namespace + `
   csi.storage.k8s.io/provisioner-secret-name: rook-csi-rbd-provisioner
-  csi.storage.k8s.io/provisioner-secret-namespace: ` + m.settings.Namespace + `
+  csi.storage.k8s.io/provisioner-secret-namespace: ` + m.settings.OperatorNamespace + `
   csi.storage.k8s.io/node-stage-secret-name: rook-csi-rbd-node
-  csi.storage.k8s.io/node-stage-secret-namespace: ` + m.settings.Namespace + `
+  csi.storage.k8s.io/node-stage-secret-namespace: ` + m.settings.OperatorNamespace + `
```

Comment:
If we have any change to the storage class, when the csi operator is enabled, we would have to delete the old storage classes and create them again. Hopefully we can avoid this upgrade headache by the csi operator being fully compatible with the storage class how it was specified by the old csi driver.

This is currently the cause of the upgrade CI test failure, since the 1.14 baseline install is attempting to use this storage class definition with the old csi driver, which of course fails.

Reply:
we should not have this change, from the functionality/user side nothing should change

Reply:
It is done because  csi operator want in resources in operator namespace and not in cephcluster namespace

Reply:
only the csi operator CR needs to be created in the operator namespace nothing like like `clusterID` `secrets` will change